deviations/viewbase.py

```diff
@@ -257,9 +257,15 @@ def get_deviation_groups(
             'submitter', 'submitter__user',
             'granter', 'granter__user',
             'exercise', 'exercise__course_module',
+            'exercise__course_module__course_instance',
         )
         # parent is prefetched because there may be multiple ancestors, and
         # they are needed for building the deviation's URL.
+        .defer(
```

Comment:
Minor nit: I think the defer call should be before the comment above, that refers to prefetch_related.